gatsby/src/pages/interactions/{BnfInteractant.slug}.module.scss

```diff
@@ -0,0 +1,129 @@
+@use '@nice-digital/nds-core/scss/core';
+
+.grid {
+  @include core.mq($from: md) {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+  }
+}
+
+.rightCol {
+  @include core.mq($from: md) {
+    flex: 0 1 33%;
+    margin-left: core.rem(core.$nds-spacing-large);
+  }
+}
+
+.informationPanel {
+  background: core.$nds-colour-nice-grey-3;
+  margin-bottom: core.rem(core.$nds-spacing-large);
+  padding: core.rem(core.$nds-spacing-medium);
+}
+
+.informationPanelHeading {
+  @include core.nds-h6();
+}
+
+.interactionInformation {
+  margin-bottom: core.rem(core.$nds-spacing-large);
+}
+
+.interactionsList {
+  list-style: none;
+  margin: 0;
+}
+
+.interactionsListItem {
+  border-bottom: 1px solid core.$nds-colour-nice-grey-1;
+
+  &:last-child {
+    border-bottom: 0;
+    margin-bottom: core.rem(core.$nds-spacing-large);
+  }
+}
+
+.filterForm {
+  @include core.mq($from: md) {
+    align-items: flex-end;
+    display: flex;
+    flex: 1 0;
+    padding-right: core.rem(core.$nds-spacing-medium);
+  }
+}
+
+.filterInput {
+  @include core.mq($from: md) {
+    flex: 1 0;
+    margin-bottom: core.rem(2px);
+  }
+}
+
+.filterButton {
+  @include core.mq($from: md) {
+    margin: 0 0 0 core.rem(core.$nds-spacing-medium);
+  }
+}
+
+.filterPanel {
+  margin-bottom: core.rem(core.$nds-spacing-medium);
+
+  @include core.mq($from: md) {
+    display: flex;
+  }
+}
+
+.clearFilterWrapper {
+  margin-bottom: core.rem(core.$nds-spacing-medium);
+}
+
+.clearFilterButton {
+  background: core.$nds-colour-nice-grey-3;
+  border: 1px solid core.$nds-colour-nice-grey-2;
```

Comment:
We have a `$nds-colour-border` which we use for _most_ borders between elements which might be more appropriate? To be fair, it's probably grey 2 under the hood!